src/main/sensors/gyro.c

```diff
@@ -625,7 +628,7 @@ void dynLpfGyroUpdate(float throttle)
             break;
         case DYN_LPF_BIQUAD:
             for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-                biquadFilterUpdateLPF(&gyro.lowpassFilter[axis].biquadFilterState, cutoffFreq, gyro.targetLooptime);
+                biquadFilterUpdateLPF(&gyro.lowpassFilter[axis].biquadFilterState, cutoffFreq, gyro.targetLooptime * dtCompensation);
```

Comment:
Q of this filter is 0.7, difference in cutoff is negligible.

Reply:
1-2% more accurate. The real benefit is just that between drones there will be slightly less variance, but on a lowpass filter the effect is much smaller than on a notch.